### PR TITLE
containerapp create response check ingress null

### PR DESCRIPTION
**Pull request by Greedygre**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your pull request is merged into main branch, a new pull request will be created to update `src/index.json` automatically.  
The precondition is to put your code inside this repository and upgrade the version in the pull request but do not modify `src/index.json`.

### CODE REVIEW

File: src/containerapp/azext_containerapp/custom.py [hunk 1]
```diff
@@ -508,7 +508,7 @@ def create_containerapp(cmd,
         if "properties" in r and "provisioningState" in r["properties"] and r["properties"]["provisioningState"].lower() == "waiting" and not no_wait:
             not disable_warnings and logger.warning('Containerapp creation in progress. Please monitor the creation using `az containerapp show -n {} -g {}`'.format(name, resource_group_name))
 
-        if "configuration" in r["properties"] and "ingress" in r["properties"]["configuration"] and "fqdn" in r["properties"]["configuration"]["ingress"]:
+        if "configuration" in r["properties"] and "ingress" in r["properties"]["configuration"] and r["properties"]["configuration"]["ingress"] and "fqdn" in r["properties"]["configuration"]["ingress"]:
```
Comment (Juliehzl) [inline src/containerapp/azext_containerapp/custom.py:511]: add test for the scenario?

Comment (Greedygre) [reply src/containerapp/azext_containerapp/custom.py:511]: ContainerappDaprTests::test_containerapp_dapr_e2e already cover case for ingress null

**Comment by Juliehzl** [comment]
update test check?

File: src/containerapp/azext_containerapp/custom.py [hunk 2]
```diff
@@ -508,7 +508,7 @@ def create_containerapp(cmd,
         if "properties" in r and "provisioningState" in r["properties"] and r["properties"]["provisioningState"].lower() == "waiting" and not no_wait:
             not disable_warnings and logger.warning('Containerapp creation in progress. Please monitor the creation using `az containerapp show -n {} -g {}`'.format(name, resource_group_name))
 
-        if "configuration" in r["properties"] and "ingress" in r["properties"]["configuration"] and "fqdn" in r["properties"]["configuration"]["ingress"]:
```
Comment (Juliehzl) [inline src/containerapp/azext_containerapp/custom.py:511]: update test for location scenario, too

Comment (Greedygre) [reply src/containerapp/azext_containerapp/custom.py:511]: Done.
test_containerapp_up_on_arc_image_e2e check environment location equals connected cluster location.

**Review by Juliehzl — APPROVED** [review]

**Pull request merged**